src/wallet/wallet2.h

```diff
@@ -1193,6 +1194,10 @@ namespace tools
     bool unblackball_output(const std::pair<uint64_t, uint64_t> &output);
     bool is_output_blackballed(const std::pair<uint64_t, uint64_t> &output) const;
 
+    // MMS -------------------------------------------------------------------------------------------------
+    mms::message_store& get_message_store() { return m_message_store; };
+    mms::multisig_wallet_state get_multisig_wallet_state();
```

Comment:
That one's probably const

Reply:
Yes. It would be bad if a function to fetch the state of the wallet would change the state ...